app/src/main/java/com/github/houseorganizer/houseorganizer/LoginActivity.java

```diff
@@ -40,8 +43,20 @@ protected void onCreate(Bundle savedInstanceState) {
         // Firebase Auth Instance
         mAuth = FirebaseAuth.getInstance();
 
-        findViewById(R.id.google_sign_in_button).setOnClickListener(
-                v -> googleSignInResultLauncher.launch(new Intent(mGoogleSignInClient.getSignInIntent())));
+        findViewById(R.id.email_signin_button).setOnClickListener(this);
+        findViewById(R.id.google_sign_in_button).setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case R.id.email_signin_button:
+                startActivity(new Intent(this, LoginEmail.class));
+                break;
+            case R.id.google_sign_in_button:
+                googleSignInResultLauncher.launch(new Intent(mGoogleSignInClient.getSignInIntent()));
+                break;
+        }
     }
```

Comment:
It might be better to use setOnClickListener for each view in onCreate, or to add an android:onClick in the xml instead of this (https://developer.android.com/guide/topics/ui/controls/button or MainScreenActivity have examples)